config/environments/production.rb

```diff
@@ -38,7 +38,9 @@
 
   # Force all access to the app over SSL and use secure cookies.
   config.force_ssl = true
-  config.ssl_options = { hsts: false }
+
+  # Set HSTS max-age to 2 years
+  config.ssl_options = { hsts: { expires: 730.days } }
```

Comment:
imho we should set this to a very small number to start, in case it does end up breaking API clients

Reply:
yeah, lets start with 5 minutes.

Reply:
Set to 5min